packages/devtools/test/timeline_protocol_test.dart

```diff
@@ -3,89 +3,794 @@
 // found in the LICENSE file.
 
 //import 'dart:async';
-//import 'dart:convert';
-//
-//import 'package:devtools/src/timeline/timeline_protocol.dart';
+import 'dart:convert';
+
+import 'package:devtools/src/timeline/timeline_protocol.dart';
+import 'package:devtools/src/utils.dart';
 import 'package:test/test.dart';
 
 void main() {
+  final originalGoldenCpuEvent = goldenCpuTimelineEvent.deepCopy();
+  final originalGoldenGpuEvent = goldenGpuTimelineEvent.deepCopy();
+  final originalGoldenCpuTraceEvents = List.of(goldenCpuTraceEvents);
+  final originalGoldenGpuTraceEvents = List.of(goldenGpuTraceEvents);
+
+  setUp(() {
+    // If any of these expect statements fail, a golden was modified while the
+    // tests were running. Do not modify the goldens. Instead, make a copy and
+    // modify the copy.
+    expect(goldenCpuString() == originalGoldenCpuEvent.toString(), isTrue);
```

Comment:
nit: make these asserts instead of expect. These aren't things we have having the test verify but invariants that make the test bogus if they are false.

Reply:
I originally had asserts, but when those failed they did not make the test fail.